### PR TITLE
Tic-Tac-Toe Project

### PR DESCRIPTION
**Pull request by Tchomp**
This is the final draft of the project.

### CODE REVIEW

File: ticTacToe.py [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
+        if turn == 'X': #If its X turn 
+            turn = 'O' #changes to O's turn
         else:
-            turn = 'X'
+            turn = 'X' #otherwise its X's turn
         
-    printBoard(board)
-    
\ No newline at end of file
+    print Board(board)
```
Comment (halfaro1) [inline ticTacToe.py]: @Tchomp why did you change this line from the original program? this should be causing an error when you run the program on your computer

Comment (Tchomp) [inline ticTacToe.py]: This was for the final draft. Originally I had something different, but errors were still being scripted. When I kept getting the same error and was told it was fine, I was posted, and found a new error. I was then told by another student that this was a common error so I kept the file as was.

Comment (halfaro1) [inline ticTacToe.py]: @Tchomp alright but line 42 as it's written is not in proper syntax. The name of the function you need to call is `printBoard` not `print Board`

Comment (Tchomp) [inline ticTacToe.py]: Alrighty sir. Thank you for the information. Do you want me to update it one more time? Or is this alright?


________________________________

Comment (halfaro1) [inline ticTacToe.py]: @Tchomp well you should correct it if you want to receive the most credit. If you want me to grade it as-is I can but you won't get a good score. Let me know how you'd like to move forward, I'm also happy to help if you need my assistance.

Comment (Tchomp) [inline ticTacToe.py]: I'll simply correct it. Thank you for looking at my work even though its late. I'm glad that I have the chance to review my errors.

________________________________

**Comment by halfaro1** [comment]
@Tchomp no problem, write a comment and at-mention me when you're done on this pull request so I can re-grade.

File: ticTacToe.py [hunk 2]
```diff
@@ -1,48 +1,39 @@
...
+    return ((board['top-L'] == player and board['top-M'] == player and board['top-R'] == player) or
+            (board['mid-L'] == player and board['mid-M'] == player and board['mid-R'] == player) or
+            (board['low-L'] == player and board['low-M'] == player and board['low-R'] == player) or
+            (board['top-L'] == player and board['mid-L'] == player and board['low-L'] == player) or
+            (board['top-M'] == player and board['mid-M'] == player and board['low-M'] == player) or
+            (board['top-R'] == player and board['mid-R'] == player and board['low-R'] == player) or
+            (board['top-L'] == player and board['mid-M'] == player and board['low-R'] == player) or
+            (board['top-R'] == player and board['mid-M'] == player and board['low-L'] == player
+             
 def startGame(startingPlayer, board):
```
Comment (halfaro1) [inline ticTacToe.py]: @Tchomp if you take a look at the failing test, you'll see that it's pointing to this line (line 21) as an error. There's actually nothing wrong with this line, but the previous line (line 19) is missing a closing parentheses which is causing the error. See it?

Comment (Tchomp) [inline ticTacToe.py]: Yes, I see it and immediately fixed it. Syntax errors are still coming up though about line 21 once I closed it with parentheses though, it keeps pointing out line 3 and 21.

**Comment by halfaro1** [comment]
@Tchomp there's still a parentheses mismatch -- have you looked through your parentheses? see how there's an opening parentheses in the series of if-statements that doesn't have a matching closing parentheses.

**Comment by halfaro1** [comment]
@Tchomp 
Thanks for continuing to work on this project! Unfortunately, I have to grade this project as-is. I am closing out this pull request and entering your grade on Blackboard. Please see your grade there.
